validations/validations.go

```diff
@@ -2,15 +2,20 @@ package validations
 
 import (
 	"fmt"
+	"io/ioutil"
+	"os"
 	"regexp"
 	"strings"
 
 	"github.com/Betterment/testtrack-cli/serializers"
+	"gopkg.in/yaml.v2"
 )
 
 const appVersionMaxLength = 18 // This conforms to iOS version numering rules
 const splitMaxLength = 128     // This is arbitrary but way bigger than you need and smaller than the column will fit
 
+const defaultOwnershipFilename = "testtrack/owners.yml"
```

Comment:
I'm new to go -- got a bit stuck trying to figure out how to make this a shared variable between `validations.go` and `validations_test.go`

Reply:
I think you'd need to capitalize it to make it visible to the test file (by default tests in go can only access public stuff to e.g. drive you toward testing your behavior not your implementation). I honestly think that I'd just spell this constant value out in the test body because it's actually defined behavior that somebody should think twice about changing.

Reply:
TY